app/models/appointment_summary.rb

```diff
@@ -32,7 +32,8 @@ def postcode=(str)
   has_many :appointment_summaries_batches, dependent: :destroy
 
   validates :title, presence: true, inclusion: { in: TITLES, allow_blank: true }
-  validates :last_name, presence: true
+  validates :first_name, length: { maximum: 50 }
+  validates :last_name, presence: true, length: { maximum: 50 }
```

Comment:
Yeah, its `OutputDocument#attendee_name` that can't be more than 50. Or perhaps more specifically, no individual line of `OutputDocument#attendee_address`